src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -38,29 +71,42 @@ import braintreeDepositAndCheckout from '@/graphql/mutation/braintreeDepositAndC
 import KvButton from '@/components/Kv/KvButton';
 import KvIcon from '@/components/Kv/KvIcon';
 import BraintreeDropInInterface from '@/components/Payment/BraintreeDropInInterface';
+import KvCheckbox from '@/components/Kv/KvCheckbox';
+import { validationMixin } from 'vuelidate';
+import { email } from 'vuelidate/lib/validators';
 
 export default {
 	components: {
 		KvButton,
 		KvIcon,
-		BraintreeDropInInterface
+		BraintreeDropInInterface,
+		KvCheckbox
 	},
 	inject: ['apollo', 'cookieStore'],
 	mixins: [
-		checkoutUtils
+		checkoutUtils,
+		validationMixin
 	],
 	props: {
 		amount: {
 			type: String,
 			default: ''
+		},
+		isGuestCheckout: {
+			type: Boolean,
+			default: true
 		}
 	},
 	data() {
 		return {
+			email: null,
 			enableCheckoutButton: false,
 			paymentTypes: ['paypal', 'card', 'applePay', 'googlePay'],
 		};
 	},
+	validations: {
+		email: { email }
+	},
```

Comment:
I think you should also add the "required" validator for the email input and the TOU checkbox.